practicesharp/Numeric_Base.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+    public interface Numeric_Base
+    {
+        private static string field;
+        public bool Validate(string number);
+        public double To_Decimal(string number);
+        public void print(double number);
+    }
+
+
+    class Binary : Numeric_Base
```

Comment:
You duplicated the code 4 times for each numeric base, why not just have one class which received the "weight" and "field" (which should be named "numericBaseCharacters"?) from the constructor?